string-manipulation-2/src/first-chars.js

```diff
@@ -0,0 +1,18 @@
+/* exported firstChars */
+function firstChars(length, string) {
+  let result = '';
+  if (string === '') {
+    return result;
+  }
+
+  if (string.length < length) {
+    for (let i = 0; i < string.length; i++) {
+      result += string[i];
+    }
+  } else if (string.length >= length) {
+    for (let i = 0; i < length; i++) {
+      result += string[i];
+    }
+  }
+  return result;
+}
```

Comment:
We can simplify this using the `slice` method
```
function firstChars(length, string) {
  return string.slice(0, length);
}
```

Reply:
Updated! Thank you!